### ISSUE TITLE
[LC] Loose ends

### ISSUE THREAD

**Issue by leram84** [tidusjar/Ombi#2124]
I'm sure there will be more "requests", but im fairly confident this is the last set of issues i could use before i actually release omBlur! @anojht has already done most of these (THANK YOU!!), but wanted to put it all down so we're all on the same page, and you can yell at us if any of this is out of line lol:

1)  The "report issue" button on the tv requests page is buried in the "view" drop down button. I need that to be on the same level as the view button/root folder/quality profile buttons, which is how it is for non admin users. And it makes sense since that is a "show wide" feature anyway. 

2) Would it be possible to add a NEW "remove" button on that level as well? movies request page has the "remove" button there, but in tv its tied to each specific request... so if 2 people request different eps of a show, you can remove each persons request one by one... thats great, but also having a remove button on the tile itself before you click the "view" dropdown would be a fast way to just clear ALL requests for that show (and would make it look like movies). 

3) The new issues details page has an id... but the id has the issue number in it, so basically theres no way to use it as an id... I need the id to be generated in a way that makes it the same for all issues.

4) 2 things from the last pr... i see the "active" classes @anojht added for the headers on the user management page and the sort dropdown on the requests page, but not for the headers on the "issues" page  or the "filter" dropdown on the requests page.

5) Can we update font-awesome and also add glyphicon-halflings to ombi? There are a few glyphs i want to use and a bunch of font-awesome ones that aren't included in whatever the current version ombi has.

6) Can we add a default poster to every item throughout ombi so if no poster loads it will show the default at least instead? `@odrel` has generously created these:
Movies: https://cdn.discordapp.com/attachments/374757488005873667/431539170268282881/Unknown-movie_notransparency.png
TV: https://cdn.discordapp.com/attachments/374757488005873667/431539008791904267/Unknown-tv_notransparency.png

7) Ombi returns a lot of junk result (just a description, and no other info, or just the release date and nothing else)... i think it would be good to have a condition that if there is no title, then dont show the result. Theres no point in requesting something with no title so would look a lot cleaner to just remove those results. If that's too much, Adding a placeholder title like `No Title` would be very helpful in keeping the css from getting all messed up when the title is absent (same issue i was having w missing posters)

8a) The Report Issue... issue - This is the most ambitious, but it will allow me to create an actual feature for ombi that i actually think MOST users will prefer. But it will only work if we can find a way to not show the "report issue" button when the "available" tag is present. Here's my ideal scenario. tell me if this can be done: right now, tv shows have the "available" tag for stuff that it has already downloaded, but no other tags (the way movies have Pending, processing request, etc). i would like to see 2 more tags added for tv. 1 for "unavailable" (when the item is not in sonarr at all), and 1 for "Pending" (for anything that exists in sonarr, but doesn't already have the "available" tag)... that would cover every tv show, so every show would either be "available", "unavailable", or "pending". THEN i also need the "report issue" button to ONLY display when the "available" or "pending" tag is present (and be removed when the "unavailable" tag is present). 

8b) If that IS all possible, i would like the "available", "pending", or "unavailable" tag to also show on the tv show tile on the tv requests page (on the same level as the "report issue" button from `#1` ^ - so you see it BEFORE you click the "view" dropdown button)... 

If all that is possible i can make an option so that users will only have the ability to request complete tv shows and not even see the options for dealing w specific episodes (though the admin will still have full control).

**Comment by tidusjar** [comment]
2. This is not a simple change, I understand your want to make things consistant like the movies page, but movies and tv shows are very different from a request point of view.
I don't think we should do this yet. If we do plan to do it then we should atleast warn the user that all child requests will be deleted.

5. Can't you just import it into your stylesheet? I don't plan to change Ombi away from Font awesome.

@leram84 We have already previously discussed 8b.

I don't think this is a good idea as it's very misleading.

If we have 3 requests for 1 show e.g. Game of thrones, request for season 1, request for season 2 and a request for season 3 to make it simple.

If Season 1 becomes availble then that's 1 request that has been fufilled. 

If we do what you are proposing then that series on the requests page will show as available which is incorrect. It could be in the state that season 1 is available, season 2 is processing and season 3 is not yet approved.

This is not a good idea and it should not be done.

Regarding 8a.
This is facing the same issue as I have highlighted above.

**Comment by leram84** [comment]
Yo! ok, about 8a/b... I wasn't trying to re-request the same thing. I thought it was a technical issue, not a design choice, and anojht seemed optimistic that it could work after we worked out some of the flaws in the concept. As far as design goes, a couple things: First of all, in the example you outlined, GoT would not show as "available" it would show as "Pending" (for the show level), and each individual ep would show whatever tag they currently have (no change there). What im proposing is absolutely ZERO change to the way the "available" tag is generated... ONLY to add an "unavailable" tag for EVERYTHING that is not in sonarr, and a "pending" tag for everything else (everything that IS in sonarr, but doesn't already have an "available" tag)... If anything this only adds two more levels of clarity, but doesn't actually change any existing rules, so i figured it should be a good design choice, just not sure how hard it will be to implement. As for the "Report Issues" button, again, not sure what it will take to code, but if the all that ^ is possible, then i would imagine it would be trivial to make the "report issues" button ONLY show when the "available" tag or the "pending" tag is present, but NOT when the "unavailable" tag is there. 

Just to be clear, the only change in the end of all this ^ would be 2 extra tags (unavailable/pending) that only show where currently nothing is displayed, and the removal of the "report issues" button for shows that aren't in sonarr. 

2) No worries. not a big deal.

3) (5 from my list) yeah, this is kind of over my head, but i couldn't figure out how to import it from css... i thought it needed to be hosted somehow. But i wasn't asking to remove font-awesome, just to UPGRADE the version of it... I THINK that was the problem. but there are a lot of font-awesome icons that i can't use and when i had that issue in org, we were able to fix it by upgrading to a more current version and then all the icons became available. ALSO, glyphicon halflings is actually there now! I don't know when it was added, but i just tested one last night and it worked! so that parts all good. Anyway, this wasn't meant to be a big thing, so don't worry about it. I thought it was literally gonna be a matter of just changing a version number on your end lol.
